config-windows.md

```diff
@@ -19,6 +19,32 @@ The Windows container specification uses APIs provided by the Windows Host Compu
     }
 ```
 
+## <a name="configWindowsDevices" />Devices
+
+**`devices`** (array of objects, OPTIONAL) lists devices that MUST be available in the container.
+
+Each entry has the following structure:
+
+* **`id`** *(string, REQUIRED)* - specifies the device for which the runtime must make available in the container.
+* **`id_type`** *(string, REQUIRED)* - tells the runtime how to interpret `id`. Today, Windows only supports a value of `class`, which identifies `id` as a device class GUID.
+
+### Example
+
+```json
+    "devices": [
+        {
+            "id": "24E552D7-6523-47F7-A647-D3465BF1F5CA",
+            "id_type": "class"
+        },
+        {
+            "id": "5175d334-c371-4806-b3ba-71fd53c9258d",
+            "id_type": "class"
+        }
+    ]
+```
+
+All devices on the host that ascribe to the declared device class GUID interface will be made available in the container. 
```

Comment:
This line seems redundant with the earlier "MUST be available in the container".  Can we drop this line?

Reply:
Yes, that sounds reasonable.